seqr/views/apis/igv_api_tests.py

```diff
@@ -292,8 +293,22 @@ def test_add_alignment_sample(self, mock_local_file_exists, mock_subprocess):
         }))
         self.assertEqual(response.status_code, 200)
 
+        # Test External AnVIL projects
+        ext_anvil_edit_url = reverse(update_individual_igv_sample, args=['I000019_na21987'])
+        self.login_collaborator()
+        response = self.client.post(ext_anvil_edit_url, content_type='application/json', data=json.dumps({
+            'filePath': '/readviz/NA21987.cram',
+        }))
+        self.assertEqual(response.status_code, 403)
+
+        self.login_manager()
+        response = self.client.post(ext_anvil_edit_url, content_type='application/json', data=json.dumps({
+            'filePath': '/readviz/NA21987.cram',
+        }))
+        self.assertEqual(response.status_code, 200)
+
```

Comment:
if it's not already covered, it would be nice if there was a test case where the requesting user isn't a pm or manager but is a collaborator that passes the `external_anvil_project_can_edit` check

Reply:
This test is testing that a non-PM user is able to successfully access this endpoint for an external anvil project. Are there any other edge cases you would want to make sure are covered?

Reply:
ah never mind then, I mistook the manager_user to be the more privileged data_manager_user.